<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryWindow</class>
 <widget class="QWidget" name="QueryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Query</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>30</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QToolButton" name="btnAddRast">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Add raster layer</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton { /* all types of tool button */
     border: 2px solid #8f8f91;
     border-radius: 6px;
     font: bold; 
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton:checked {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 white, stop: 1 rgb(108, 183, 255) );
 }

 /* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 2px solid gray;
     border-top-right-radius: 6px;
     border-bottom-right-radius: 6px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow {
     image: url(downarrow.png);
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="icon">
           <iconset resource="../gui/resources.qrc">
            <normaloff>:/icons/icons/element-cell.gif</normaloff>:/icons/icons/element-cell.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDelRast">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Remove Raster Layer</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton { /* all types of tool button */
     border: 2px solid #8f8f91;
     border-radius: 6px;
     font: bold; 
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton:checked {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 white, stop: 1 rgb(108, 183, 255) );
 }

 /* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 2px solid gray;
     border-top-right-radius: 6px;
     border-bottom-right-radius: 6px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow {
     image: url(downarrow.png);
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="icon">
           <iconset resource="../gui/resources.qrc">
            <normaloff>:/icons/icons/element-cell-delete.gif</normaloff>:/icons/icons/element-cell-delete.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="refreshlayerlist">
          <property name="styleSheet">
           <string notr="true">QToolButton { /* all types of tool button */
     border: 2px solid #8f8f91;
     border-radius: 6px;
     font: bold; 
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton:checked {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 white, stop: 1 rgb(108, 183, 255) );
 }

 /* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 2px solid gray;
     border-top-right-radius: 6px;
     border-bottom-right-radius: 6px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow {
     image: url(downarrow.png);
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../gui/resources.qrc">
            <normaloff>:/icons/icons/mActionDraw.png</normaloff>:/icons/icons/mActionDraw.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnAddVect">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Add Vector Layer</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton { /* all types of tool button */
     border: 2px solid #8f8f91;
     border-radius: 6px;
     font: bold; 
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton:checked {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 white, stop: 1 rgb(108, 183, 255) );
 }

 /* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 2px solid gray;
     border-top-right-radius: 6px;
     border-bottom-right-radius: 6px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow {
     image: url(downarrow.png);
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="icon">
           <iconset resource="../gui/resources.qrc">
            <normaloff>:/icons/icons/element-vector.gif</normaloff>:/icons/icons/element-vector.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDelVect">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Remove Vector Layer</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton { /* all types of tool button */
     border: 2px solid #8f8f91;
     border-radius: 6px;
     font: bold; 
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton:checked {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 white, stop: 1 rgb(108, 183, 255) );
 }

 /* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 2px solid gray;
     border-top-right-radius: 6px;
     border-bottom-right-radius: 6px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow {
     image: url(downarrow.png);
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="icon">
           <iconset resource="../gui/resources.qrc">
            <normaloff>:/icons/icons/element-vector-remove.gif</normaloff>:/icons/icons/element-vector-remove.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="renderoptions">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton { /* all types of tool button */
     border: 2px solid #8f8f91;
     border-radius: 6px;
     font: bold; 
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton:checked {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 white, stop: 1 rgb(108, 183, 255) );
 }

 /* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 2px solid gray;
     border-top-right-radius: 6px;
     border-bottom-right-radius: 6px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow {
     image: url(downarrow.png);
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
          </property>
          <property name="text">
           <string>R</string>
          </property>
          <property name="icon">
           <iconset resource="../gui/resources.qrc">
            <normaloff>:/icons/icons/gimp_0.png</normaloff>:/icons/icons/gimp_0.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QToolButton" name="setPenColor">
          <property name="styleSheet">
           <string notr="true">QToolButton { /* all types of tool button */
     border: 2px solid #8f8f91;
     border-radius: 6px;
     font: bold; 
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton:checked {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 white, stop: 1 rgb(108, 183, 255) );
 }

 /* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 2px solid gray;
     border-top-right-radius: 6px;
     border-bottom-right-radius: 6px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow {
     image: url(downarrow.png);
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
          </property>
          <property name="text">
           <string>Pen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PenColor">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>111,111,111</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="setBrushColor">
          <property name="styleSheet">
           <string notr="true">QToolButton { /* all types of tool button */
     border: 2px solid #8f8f91;
     border-radius: 6px;
     font: bold; 
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton:checked {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 white, stop: 1 rgb(108, 183, 255) );
 }

 /* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 2px solid gray;
     border-top-right-radius: 6px;
     border-bottom-right-radius: 6px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow {
     image: url(downarrow.png);
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
          </property>
          <property name="text">
           <string>Brush</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="BrushColor">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>111,111,111</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PointSize">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>1,1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="LineWidth">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Fill">
          <property name="text">
           <string>Fill</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Thickness">
          <property name="text">
           <string>Thickness</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>kml</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTableWidget" name="layerTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string>GRASS Layer(s)</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="getlisttoquery">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton { /* all types of tool button */
     border: 2px solid #8f8f91;
     border-radius: 6px;
     font: bold; 
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton:checked {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 white, stop: 1 rgb(108, 183, 255) );
 }

 /* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 2px solid gray;
     border-top-right-radius: 6px;
     border-bottom-right-radius: 6px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow {
     image: url(downarrow.png);
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/gui-help.gif</normaloff>:/icons/gui-help.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="update">
          <property name="styleSheet">
           <string notr="true">QToolButton { /* all types of tool button */
     border: 2px solid #8f8f91;
     border-radius: 6px;
     font: bold; 
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton:checked {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 white, stop: 1 rgb(108, 183, 255) );
 }

 /* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 2px solid gray;
     border-top-right-radius: 6px;
     border-bottom-right-radius: 6px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow {
     image: url(downarrow.png);
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/grass_icon.png</normaloff>:/icons/grass_icon.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="longitude">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
     border: 1px solid gray;
     border-radius: 5px;
     padding: 0 8px;
     background: rgb(231, 231, 231);
 }</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="latitude">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
     border: 1px solid gray;
     border-radius: 5px;
     padding: 0 8px;
     background: rgb(231, 231, 231);
 }</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="gcmd"/>
        </item>
        <item>
         <widget class="QToolButton" name="gcmdexec">
          <property name="styleSheet">
           <string notr="true">QToolButton { /* all types of tool button */
     border: 2px solid #8f8f91;
     border-radius: 6px;
     font: bold; 
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
     padding-right: 20px; /* make way for the popup button */
 }

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton:checked {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 white, stop: 1 rgb(108, 183, 255) );
 }

 /* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 2px solid gray;
     border-top-right-radius: 6px;
     border-bottom-right-radius: 6px;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow {
     image: url(downarrow.png);
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/grass_icon.png</normaloff>:/icons/grass_icon.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../gui/resources.qrc"/>
 </resources>
 <connections/>
</ui>
